Packages/com.unity.inputsystem/InputSystem/Devices/Keyboard.cs

```diff
@@ -1971,7 +2038,7 @@ public string keyboardLayout
         /// True when IME composition is enabled.  Requires <see cref="Keyboard.SetIMEEnabled"/> to be called to enable IME, and the user to enable it at the OS level.
         /// </summary>
         /// <remarks>
-        ///
+        /// <see cref="ButtonControl"/> representing a combined left and right alt key.
```

Comment:
Do we need this remark? I don't quite understand why this is relevant for `imeSelected`?

Reply:
fixed in [8ad97b6](https://github.com/Unity-Technologies/InputSystem/pull/2069/commits/8ad97b6e1a384f347459dbfa06ba72c998464ce4)